ValueChangedGenerator/ValueChangedGenerator/ValueChangedGenerator.SourceGenerator/RoslynExtensions.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Collections.Generic;
```

Comment:
How about using the following `Compile` tags instead of copy-and-pasting files

```xml
  <ItemGroup>
    <Compile Include="../ValueChangedGenerator.CodeFixes/RecordDefinition.cs" />
    <Compile Include="../ValueChangedGenerator.CodeFixes/SyntaxExtensions.cs" />
  </ItemGroup>
```